qfdmo/models/change_suggestion.py

```diff
@@ -0,0 +1,126 @@
+"""
+Modèle de données Django pour les suggestions de changements
+"""
+
+from textwrap import dedent
+
+from django.db import models
+from django.utils import timezone
+
+
+class ChangeSuggestion(models.Model):
+    """
+    Modèle de données Django pour les suggestions de changements
+
+    Gestion des données:
+     - change_data: ne doit contenir que le strict minimum pour gérer le changement
+        (ex: pour un cluster d'acteur = cluster_id -> acteur_id). Ne pas stocker
+        d'autres données qui pourraient devenir obsolètes au moment de la revue.
+        Si on souhaite fournir de la donnée supplémentaire, il faut le faire à
+        la vollée (ex: au moment de l'affichage ou de l'export)
+
+     - debug_data (optionnel): si on veut expliquer comment on est arrivé à cette
+        suggestion, on peut stocker ici des données de debug (ex: valeurs des champs,
+        scores de similarité, etc.)
+    """
+
+    class Meta:
+        db_table = "qfdmo_change_suggestions"
+        verbose_name = "Suggestion de changement"
+        verbose_name_plural = "Suggestions de changements"
+
+    # ------------------------------------
+    # Champs liés à la suggestion
+    # ------------------------------------
+    id = models.AutoField(primary_key=True)
+    task_run_id = models.CharField(
+        max_length=200, verbose_name="ID de la tâche qui a généré la suggestion"
+    )
+    task_run_at = models.DateTimeField(
+        verbose_name="Date/heure de la tâche qui a généré la suggestion"
+    )
+    debug_data = models.JSONField(
+        verbose_name=dedent(
+            """Données de debug de la suggestion
+        (quelles données ont été utilisées pour la suggestion)"""
+        ),
```

Comment:
```suggestion
        verbose_name="Données de debug de la suggestion"
        "(quelles données ont été utilisées pour la suggestion)",
```

python permet d'écrire des strings sur plusieurs lignes dans ce genre de situations